Dockerfile.cpu

```diff
@@ -6,7 +6,16 @@ RUN apt-get update  -y \
     && apt-get install -y git wget vim numactl gcc-12 g++-12 python3 python3-pip libtcmalloc-minimal4 \
     && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-12 10 --slave /usr/bin/g++ g++ /usr/bin/g++-12
 
-RUN echo 'export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libtcmalloc_minimal.so.4:$LD_PRELOAD' >> ~/.bashrc
+RUN pip install accelerate mkl
```

Comment:
QQ: Where are these libraries used?

Reply:
I actually use `accelerate` the setup the notebook but indeed it's not necessary here - just removed the `accelerate` and `mkl` package, and install `intel-openmp` only here